### ISSUE TITLE
Selected State does not annonce when Slider Component selected

### ISSUE THREAD

**Issue by amarlette** [facebook/react-native#30956]
## Description
Does not announce "selected" on the focus of Slider Component.

This issue is covered in-depth by parent issue #30842, please check there for more details.

## React Native version:
0.63

**Comment by meehawk** [comment]
Just curious if this is still relevant since the Slider component seems to have been deprecated as per [#30945](https://github.com/facebook/react-native/issues/30945)

**Comment by amarlette** [comment]
@meehawk Thank you for tagging me on this issue, I'll see if this is still relevant.

@kacieb or @lunaleaps since this is a deprecated component per https://reactnative.dev/docs/slider can we close #30956 and #30945?

**Comment by lunaleaps** [comment]
@amarlette Since we have https://github.com/facebook/react-native/pull/31145, let's keep this open for that but we can close #30945 since it's not relevant to the community

**Comment by amarlette** [comment]
Thank you @sladyn98 for your contribution to a more accessible React Native! Can you comment your Twitter handle if you have one so that I can give you a shout-out?

**Comment by sladyn98** [comment]
@amarlette my twitter handle is [@SladynN](https://twitter.com/SladynN) .Thanks for the shoutout

**Comment by gabrieldonadel** [comment]
Quick question @amarlette and @lunaleaps, is there something else to be done here or can we close this issue?

**Comment by lunaleaps** [comment]
I _think_ we can close this issue, thanks for the notice!
